servicex/__init__.py

```diff
@@ -71,12 +96,42 @@ def create_app(test_config=None,
     Bootstrap5(app)
 
     JWTManager(app)
+
+    # setup logging
+    instance = os.environ.get('INSTANCE_NAME', 'Unknown')
+    formatter = ServiceXFormatter('%(levelname)s ' +
+                                  f"{instance} servicex_app " +
+                                  '%(requestId)s %(message)s')
+    servicex_handler = logging.StreamHandler()
+    servicex_handler.setFormatter(formatter)
+    if app.debug:
+        servicex_handler.setLevel('DEBUG')
+        app.logger.level = logging.DEBUG
+    else:
+        levels = {
```

Comment:
Looks like `setLevel` [accepts strings](https://docs.python.org/3/library/logging.html#logging.Logger.setLevel) so this dict is not needed

Reply:
I made a slight change to fix a related bug but I still need to set app.logger.level based on the env so it's either using the dict or a bunch of if/else statements.